external/storm-kafka-client/src/test/java/org/apache/storm/kafka/spout/trident/KafkaTridentSpoutEmitterEmitTest.java

```diff
@@ -26,11 +26,12 @@
 import static org.mockito.Mockito.when;
 
 import java.util.Collections;
+import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
-import org.apache.kafka.clients.consumer.ConsumerRecord;
-import org.apache.kafka.clients.consumer.MockConsumer;
-import org.apache.kafka.clients.consumer.OffsetResetStrategy;
+
+import org.apache.commons.collections.map.HashedMap;
+import org.apache.kafka.clients.consumer.*;
```

Comment:
Nit: Please avoid wildcard imports, they make the code harder to read

Reply:
🤦‍♂